front-end/src/components/Card/Reply.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { WatchLater } from '@material-ui/icons';
+import { GridContainer, GridItem } from '../Grid';
+import { Text } from '../Text';
+
+export const Reply = ({ replyId, comments, users }) => {
+  return (
+    <GridItem xs={12}>
+      <div
+        style={{
```

Comment:
Remove styles wherever you used and in elements and use styled-components to styles

Reply:
Replaced all style props with styled components and component props where needed.